UnitySDK/Assets/ML-Agents/Scripts/SideChannel/FloatPropertiesChannel.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using System.IO;
+using System;
+using System.Text;
+
+namespace MLAgents
+{
+
+    public interface IFloatProperties
+    {
+        void SetProperty(string key, float value);
+        float GetPropertyWithDefault(string key, float defaultValue = 0f);
```

Comment:
YES

Reply:
Yes to what? Is there something that is also applicable to other side channels?

Reply:
The "WithDefault" part :)  Probably not applicable elsewhere.